README.md

```diff
@@ -1,8 +1,8 @@
-<img width="150" align="left" src="Source/Hurl.BrowserSelector/Assets/internet.ico">
+# Hurl
 
-<h1 align="center">Hurl</h1>
+A Windows utility that lets you choose a browser on the click of a link.
 
-<p align="center">A windows utility that lets you choose a browser on the click of a link</p>
+![Hurl Icon](Source/Hurl.BrowserSelector/Assets/internet.ico)
```

Comment:
I prefer the Icon/Logo is either centered or in a two column layout as it was previously.

So you can leave it as it was earlier. but open to any suggestions here

Reply:
Okay, a centered `<p>` should be enough.

Reply:
By default, all text in Markdown fields are aligned to the left. Looks like a Markdown image inside HTML (either `p` or `div`) isn't the way. Compare these two:
![image](https://github.com/user-attachments/assets/39724cd4-3406-4a59-b993-93bf3feb2f09)
and
![image](https://github.com/user-attachments/assets/3167f3fc-9efa-49b9-9052-6cea310bc5fc)

Notice how hovering over the second URI doesn't offer to open the file.

Reply:
Let's currently go with 3 (I made additional few tweaks) for now. And will come back to 2. later once we have a fresh icon for hurl